theseus/embodied/collision/collision.py

```diff
@@ -32,7 +32,12 @@ def __init__(
         self.sdf_origin = sdf_origin
         self.sdf_data = sdf_data
         self.sdf_cell_size = sdf_cell_size
-        self.cost_eps = cost_eps
+        if not isinstance(cost_eps, Variable):
+            if not isinstance(cost_eps, torch.Tensor):
+                cost_eps = torch.tensor(cost_eps)
```

Comment:
Likely need to add batch size here (and in rest of cost functions) same as pose.

Reply:
Thanks for clarifying Luis. @jeffin07, the batch size here you can get from the passed pose object.
Nice catch on the bug as well, Luis.

Reply:
It's probably better to use `(-1, 1)` instead of getting the batch size from the pose object, because `dt` is an element that we typically set a batch size of 1 for in our examples, regardless of the pose batch.